source/sass/blocks/page-nav.scss

```
@@ -1,35 +1,124 @@
-.page-nav__toggle {
-  position: absolute;
-  padding: 0;
-  right: -22px;
-  top: -12px;
-  width: 60px;
-  height: 60px;
-  border: none;
-  background-color: transparent;
-  cursor: pointer;
-  z-index: 2;
-
-  @media (min-width: $vp-768) {
+@media (max-width: $vp-767) {
+  .page-nav--closed .page-nav__list {
     display: none;
   }
 }
 
+// .page-nav--nojs + .page-nav__toggle {
+//   display: none;
+// }
+
+// .page-nav--nojs + .page-nav__list {
+//   position: static;
+//   min-height: 0;
+// }
+
 .page-nav__list {
   list-style-type: none;
   margin: 0;
   padding: 0;
   display: flex;
   flex-direction: row;
+
+  @media (max-width: $vp-767) {
+    position: absolute;
+    top: 0;
+    right: 0;
+    flex-direction: column;
+    color: $color-tangaroa;
+    background-color: $color-alice-blue;
+    padding: 75px 30px 30px 15px;
+    width: 285px;
```

Чтобы не пересчитывать тень
```suggestion
    width: calc(100vw - 35px);
```